libsplinter/src/admin/store/yaml/mod.rs

```diff
@@ -1130,7 +1131,14 @@ impl TryFrom<YamlCircuitState> for CircuitState {
 
     fn try_from(state: YamlCircuitState) -> Result<Self, Self::Error> {
         Ok(CircuitState {
-            nodes: state.nodes,
+            nodes: state
+                .nodes
+                .into_iter()
+                .map(|(id, node)| match CircuitNode::try_from(node) {
```

Comment:
This `match` looks like it can be replaced with a `.map()`

Reply:
Changed to use `.map()`